packages/subgraph/src/wPowerPerp.ts

```diff
@@ -0,0 +1,22 @@
+import { BigDecimal } from "@graphprotocol/graph-ts";
+import { Transfer } from "../generated/WPowerPerp/WPowerPerp";
+import { buyOrSellSQTH, createTransactionHistory } from "./util";
+
+export function handleTransfer(event: Transfer): void {
+  let senderTransactionHistory = createTransactionHistory("SEND_OSQTH", event);
+  senderTransactionHistory.owner = event.params.from.toHex();
+  senderTransactionHistory.oSqthAmount = event.params.value;
+  senderTransactionHistory.save();
+
+  let receiverTransactionHistory = createTransactionHistory(
```

Comment:
it's a bit strange to trigger receive_osqth event with sender address using event.params.from.toHex(), cuz it only means sending osqth history to them